src/components/Header/Header.js

```diff
@@ -0,0 +1,26 @@
+import { Link } from "react-router-dom";
+import { useGameContext } from "../GameProvider/GameProvider";
+import { Row } from "../Row/Row";
+import style from "./_header.module.css";
+
+export function Header() {
+	const gameContext = useGameContext();
+	
+	const rounds = gameContext.historyHits.filter((p) => {
+		return p.playerId === gameContext.historyHits[0].playerId;
+	});
+
+	return (
+		<Row className={style.header}>
+			{/* <Logo className={style.logo} /> */}
```

Comment:
remove comment and class if it's not used here anymore

Reply:
Removed